src/dreyfus_index_manager.erl

```diff
@@ -86,6 +86,12 @@ handle_call({open_error, DbName, Sig, Error}, {OpenerPid, _}, State) ->
 
 handle_cast({cleanup, DbName}, State) ->
     clouseau_rpc:cleanup(DbName),
+    {noreply, State};
+
+handle_cast({move, DbName}, State) ->
+    RecoveryDir = config:get("couchdb", "recovery_index_dir", ".recovery"),
```

Comment:
the recovery dir should be specified as an absolute path. I think that needs to happen in the dbcore chef cookbook. It should be under the `[:clouseau][:index_dir]` directory.